README.markdown

```diff
@@ -190,11 +190,13 @@ let colour = "red"
 
 ## Code Organization
 
-Use extensions to organize your code into logical blocks of functionality. Each extension should be set off with a `// MARK: -` comment to keep things well-organized.
+Use extensions to organize your code into logical blocks of functionality. Each extension should be set off with a `// MARK: -` comment to keep things well-organized. 
+<!-- [@skarol] - this is a little bit controversial as extensions are not meant to organize code. Moreover, grouping part of codes in extensions usually means that class/struct has too many responsibilities and this should be avoided. However having a model class/struct that has to conform to some protocol might fit here. -->
```

Comment:
We don't actually follow the rule in Lake. Good examples how we don't follow the rule are: `CardViewController` and `GridCollectionViewController` but I think separating it out would make sense, well for `CardViewController` maybe not since it might be obsolete in upcoming months (some UI/UX/design architecture changes that Goran and Lea mentioned).